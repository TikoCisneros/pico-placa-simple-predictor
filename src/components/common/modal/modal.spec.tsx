/* eslint-disable @typescript-eslint/no-empty-function */
import React from 'react';
import { render } from '@testing-library/react';
import Modal from './modal';

export const defaultProps = {
  open: true,
  title: 'Are you ready?',
  message: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
  onClose: () => {},
};

export const data = [
  'Fusce rutrum tellus at libero ultrices pulvinar.',
  'Ut viverra consequat quam, nec porttitor nunc sodales sit amet.',
  'Sed egestas molestie sem.',
];

describe('Layout test', () => {
  it('should renders correctly without items', () => {
    const { asFragment } = render(<Modal {...defaultProps} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should renders correctly with items', () => {
    const props = {
      ...defaultProps,
      items: data,
    };
    const { asFragment } = render(<Modal {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});