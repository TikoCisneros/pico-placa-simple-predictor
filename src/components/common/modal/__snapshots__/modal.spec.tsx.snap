// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout test should renders correctly with items 1`] = `<DocumentFragment />`;

exports[`Layout test should renders correctly without items 1`] = `<DocumentFragment />`;
